Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,8 +70,16 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
-            return array;
+            if (IsArrayNullOrEmpty(array)) return array;
+            if (index < 0 || index > (array.Length - 1)) return array;
```

Comment:
could simplify this or make it more readable at least (also prefer to state, rather than negate):
```
private static bool IsValidIndex(int[] array, int index) => index >= 0 || index < (array.Length)
```
and then the code becomes:
```cs
if (!IsValidIndex(array, index)) return array;
```